packages/libs/src/sdk/examples/react/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="https://cdn.jsdelivr.net/npm/web3@latest/dist/web3.min.js"></script>
```

Comment:
we need web3.min for sdk?

Reply:
Yeah, it's in the docs too. It has to do with the polyfills required inside of the web3 package 
![image](https://github.com/AudiusProject/audius-protocol/assets/19916043/cfb73268-e3b7-473e-8e46-c560cdcbf8ee)